### PR TITLE
Make Tests Windows compatible - Part II

### PR DESCRIPTION
**Pull request by aepfli**
As the first Pull-request got merged with an open discussion,
here are the proposed changes for all issues with windows
test execution.

### CODE REVIEW

**Comment by aepfli** [comment]
i found the issue

```
Condition not satisfied:

Files.getLastModifiedTime(lastUpdate).toInstant().isAfter(beforeBootstrap)
|     |                   |           |           |       |
|     2022-04-01T13:22:28Z|           |           false   2022-04-01T13:22:28.470Z
class java.nio.file.Files |           2022-04-01T13:22:28Z
                          /var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/spock_boostrap_sets_up_db_0_testProjectDir5079185309897053008/hybris/bin/platform/lib/dbdriver/.lastupdate

	at mpern.sap.commerce.build.BootstrapTest.boostrap sets up db drivers(BootstrapTest.groovy:177)
```

looks like `Files.getLasModifiedTime` is not providing milliseconds, and thats why we have this isse

**Comment by aepfli** [comment]
when you merge this Pull Request, either squash it, or let me know, i will clean the commit history up

File: src/functionalTest/groovy/mpern/sap/commerce/build/BootstrapTest.groovy [hunk 1]
```diff
@@ -156,7 +156,10 @@ class BootstrapTest extends Specification {
 
         when: "running bootstrap task"
 
-        def beforeBootstrap = Instant.now()
+        def beforeBootstrap = Instant.now() - 5
```
Comment (aepfli) [inline src/functionalTest/groovy/mpern/sap/commerce/build/BootstrapTest.groovy:159]: i set the time to 5s earlier - this way we prevent the milisecond issue - we could also try to strip and and do an is after or equals check. the question is what is more desirable

**Comment by aepfli** [comment]
sidenote this will also add the reports as action artifacts - for easier verification: https://github.com/SAP/commerce-gradle-plugin/actions/runs/2077603613 - see the overview page

File: src/functionalTest/groovy/mpern/sap/commerce/build/BootstrapTest.groovy (hunk 1 above)
Comment (mpern) [reply src/functionalTest/groovy/mpern/sap/commerce/build/BootstrapTest.groovy:159]: Lowering the precision to seconds + after equals is a bit cleaner

Comment (aepfli) [reply src/functionalTest/groovy/mpern/sap/commerce/build/BootstrapTest.groovy]: done

**Pull request merged**